Sources/PT.PM.CSharpParseTreeUst/Converter/ExpressionVisitor.cs

```diff
@@ -590,15 +590,17 @@ public override Ust VisitLiteralExpression(LiteralExpressionSyntax node)
                 case "Boolean":
                     return new BooleanLiteral((bool)token.Value, node.GetTextSpan());
                 case "Int32":
-                case "UInt32":
                 case "Int16":
                 case "UInt16":
                 case "Byte":
                 case "SByte":
+                    return new IntLiteral(System.Convert.ToInt32(token.Value), node.GetTextSpan());
+                case "UInt32":
+                    return new IntLiteral((int)System.Convert.ToUInt32(token.Value), node.GetTextSpan());
```

Comment:
Maybe we should use `LongLiteral` for `UInt32` and `BigIntLiteral` for `UInt64`. What do u think about this?

Reply:
`BigInt` feels a bit wasteful for `UInt64`...
Can we make `IntLiteral` and `LongLiteral` seamlessly handle both signed and unsigned integers under the hood?

Reply:
We describe syntactic (unsigned) integers, not calculations. Thus `IntLiteral` class can represent both just keeping two fields - for both signed and unsigned, assigning to the right one upon construction and remembering which one to use.

More OOP way is two have two subclasses of `IntLiteral` of course...